### DISCUSSION TITLE
import NextImage from 'next/image'

### DISCUSSION THREAD

**Discussion by rsipakov** [timlrx/tailwind-nextjs-starter-blog#298] [Q&A] [answered]
Hey there, 

Could you explain why we use `import NextImage from 'next/image'` in \components\Image.js? Why not just use `import Image from 'next/image'`? 

Thank you!

**Comment by timlrx** [comment, marked as answer]
That's because in the subsequent line we are defining a new image component `const Image = ({ ...rest }) => <NextImage {...rest} />` and there would be a clash in the names if the import was also called `Image`. 

As to why there is this additional level of abstraction, it's mainly to facilitate those trying to create a static build since they can swap out `NextImage` for `img`.
